test/support.js

```diff
@@ -101,14 +101,13 @@ const Support = {
 
     const config = Config[options.dialect];
 
-    const sequelizeOptions = _.defaults(options, {
-      host: options.host || config.host,
+    const sequelizeOptions = _.defaultsDeep(options, {
```

Comment:
Since _.defaultsDeep (as well as _.default) overrides the input object, I wonder if we should rather change it to:

```
const sequelizeOptions = _.merge({
      host: config.host,
      logging: process.env.SEQ_LOG ? console.log : false,
      port: process.env.SEQ_PORT || config.port,
      pool: config.pool,
      dialectOptions: config.dialectOptions || {},
      minifyAliases: config.minifyAliases
    }, options)

```

Reply:
Not 100% sure if it is producing the same object, but it projects the idea :)

Reply:
I don't think it's a problem to change the config, but sure.